manifests/config/audit_profiles.pp

```diff
@@ -64,13 +64,13 @@
     $_buffer_size = $auditd::buffer_size
   }
 
+  file { '/etc/audit/rules.d/00_head.rules':
```

Comment:
Why are you including these rules everywhere, now?

Reply:
@lnemsick-simp It turns out that not including that part of the configuration in all cases will end up with a non-functional auditd. I can put it in the changelog. These are more "global" configuration items rather than actual rules that are going to cause issues.

Reply:
Also, if you set the profile list to `[]`, then nothing is included from this file anyway.